Toolchain/serenity.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation {
     xlibsWrapper
     qemu
     e2fsprogs
-    fuse2fs
+    # fuse2fs
```

Comment:
Why would you comment this out instead of removing it?

Reply:
Because the glibc below it is commented as well and I didn't want to rock the boat, it makes things easier to note what's missing for someone on an older nixpkgs. Could easily well remove it and the commented glibc though.

Reply:
> it makes things easier to note what's missing for someone on an older nixpkgs

Then leave a proper (https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#comments) comment at least :)